### DISCUSSION TITLE
[Tests] Indentación parte 1-3

### DISCUSSION THREAD

**Discussion by jojopalma** [IIC2133-PUC/2024-1#41] [Tarea 0] [answered]
Hola 🖐🏼, 

Luego de revisar los test de la parte señalada (1-3) con el script compare.sh, me percaté de que la indentación es reconocida como distinta por el script si se utiliza el scape sequence "\t" o utilizando 4 espacios, ¿debería utilizar aquella forma que me asegura igualdad de output, o debería ser indiferente?.

Adjunto la imagen para mayor claridad
![image](https://github.com/IIC2133-PUC/2024-1/assets/69826716/bd693c7c-d7e2-4553-93e6-c8b7302c6679)

**Comment by PabloNavarroY** [comment, marked as answer]
Te recomiendo ocupar espacios, porque si no lo tienes malo.
